pyfar/dsp/dsp.py

```diff
@@ -2669,3 +2669,175 @@ def normalize(signal, reference_method='max', domain='auto',
         return normalized_signal, reference_norm
     else:
         return normalized_signal
+
+
+def correlate(signal_1, signal_2, mode='full'):
```

Comment:
Have we considered providing a `normalise` argument? 
This could be a quite nice feature when the correlation function is used as a similarity measure for two-signals.

Reply:
* normalize

Reply:
added the normalize parameter as suggested